website/src/CONTRIBUTING.md

```diff
@@ -0,0 +1,309 @@
+# Contributing to OpenEBS Website
+
+## Add a new blog post
+
+* Create a markdown file and add your blog content along with the metadata in    the following format
+```
+---
+title: Title of the Blog
+author: Author name
+author_info: Author information
+tags: All the tags with space in between (Example: openebs localpv)
+date: Date of publication in dd-mm-yyyy format
+---
+ 
+Blog content in the markdown format goes here…
+
+```
+Reference for text formatting in markdown https://guides.github.com/features/mastering-markdown/
+* Name of the markdown file should be the title of the blog converted to all small letters with hyphen(-) separation between the words.
+For example, If the title of the blog is: Title of the Blog
+Then, markdown file name will be title-of-the-blog.md 
+* Add this markdown file to website/src/blogs
+* The main blog image should be named as title-of-the-blog.png and add this file to website/public/Images/blog
```

Comment:
title-of-the-blog.png ==> highlight it please

Reply:
website/public/Images/blog ==>. this one also.

Reply:
Please go through the content and highlight the portion of the content which will be helpful to understand the context better.